### ISSUE TITLE
Sync-DbaAvailabilityGroup -Login

### ISSUE THREAD

**Issue by pasqualeceglie** [sqlcollaborative/dbatools#5683]
#### SQL Server: 

```sql
/* 
Microsoft SQL Server 2012 (SP4) (KB4018073) - 11.0.7001.0 (X64) Aug 15 2017 10:23:29 Copyright (c) Microsoft Corporation Enterprise Edition: Core-based Licensing (64-bit) on Windows NT 6.1 <X64> (Build 7601: Service Pack 1)
 */
```

#### Errors Received
Even if I specify -Login option for only one login, the command was execute for all logins in my instance

#### Expected Behavior
Only one login is synced

#### Actual Behavior
All logins are processed

**Comment by potatoqualitee** [comment]
thank you for the report, ill take a look early next week and this will be resolved for 1.0

**Comment by potatoqualitee** [comment]
I just noticed too that required information for a bug report has been left out, particularly the version of dbatools which is perhaps the most important information. Showing the exact command that is used is also helpful.

Ultimately, I am unable to repro. 

![image](https://user-images.githubusercontent.com/8278033/59151383-680a1580-8a32-11e9-99a6-2a71382e4d41.png)

If you continue to experience the same issue with the latest version of dbatools, please create a bug report with the required information. If you can include sanitized output as well, please add a -Verbose flag so I can see what's going on.
